vignettes/getting.Rmd

```diff
@@ -190,7 +190,7 @@ If you have not already done so, you will need to install the R packages we will
 ```{r, message=FALSE}
 library(pct)
 library(sf)      # key package for working with spatial vector data
-library(dplyr)   # in the tidyverse
+library(tidyverse)   # in the tidyverse
```

Comment:
Think that's what's breaking actions. I agree it would make sense for people to load the whole tidyverse rather than just `dplyr`...

Reply:
ah, I changed this as we are now using the `summarise_if` function from `tidyverse`

Reply:
`summarise_if` is in `dplyr`. had to double check but this is pretty conclusive: 

``` r
dplyr::summarise_if
#> function (.tbl, .predicate, .funs, ...) 
#> {
#>     lifecycle::signal_superseded("1.0.0", "summarise_if()", "across()")
#>     funs <- manip_if(.tbl, .predicate, .funs, enquo(.funs), caller_env(), 
#>         ..., .caller = "summarise_if")
#>     summarise(.tbl, !!!funs)
#> }
#> <bytecode: 0x558b932711a0>
#> <environment: namespace:dplyr>
```

<sup>Created on 2021-08-24 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>